backend/cdb_csv/pe.py

```diff
@@ -260,7 +260,10 @@ async def import_actions(connection: Connection, action_csv_path: str):
                 )
                 continue
 
-            focus: str | None = mapping[csv_row.lblaction]
+            try:
+                focus: str | None = mapping[csv_row.lblaction]
+            except KeyError:
+                focus = None
```

Comment:
Suggestion (pour une autre fois 😉 )
```suggestion
            focus: str | None = mapping.get(csv_row.lblaction, None)
```
Ou plus court mais peut-être moins clair, juste
```suggestion
            focus: str | None = mapping.get(csv_row.lblaction)
```